model/states.go

```diff
@@ -137,10 +166,32 @@ type OperationState struct {
 	Timeouts *OperationStateTimeout `json:"timeouts,omitempty"`
 }
 
+type operationStateForUnmarshal OperationState
+
+// UnmarshalJSON unmarshal EventState object from json bytes
+func (o *OperationState) UnmarshalJSON(data []byte) error {
+
+	var timeout OperationStateTimeout
+	if err := json.Unmarshal(data, &timeout); err != nil {
+		return err
+	}
+
+	v := operationStateForUnmarshal{
+		Timeouts: &timeout,
```

Comment:
we should keep it nil when the user didn't define it.

Reply:
indeed